### ISSUE TITLE
예제 5.8에 errata가 있네요.

### ISSUE THREAD

**Issue by choij1589** [wikibook/tf2#8]
책에 있는 코드를 하나하나 치다가 에러가 나서 keras class reference를 검색해 보니,
loss='sparse_categorical_crossentropy' 에서 sparse_categorical_crossentropy는 One-Hot encoding방식을 지원하지 않네요.

소스 코드에서는 loss='categorical_crossentropy'를 사용해서 바꿔보니 잘 돌아갑니다.

참조: https://keras.io/losses/

**Comment by greentec** [comment]
안녕하세요.
말씀주신대로 오타가 맞습니다. 코랩과 깃허브, [출판사 홈페이지의 정오표](https://wikibook.co.kr/tf2/)에는 해당 내용이 수정되어 있습니다.
감사합니다.
